docs/essentials/collections/selectingaCollectionClass.md

```diff
@@ -1 +1,69 @@
+# Selecting a Collection Class
 
+Be sure to choose your collection class carefully. Using the wrong type can restrict your use of the collection. In general, avoid using the types in the System.Collections namespace unless you are specifically targeting .NET Framework version 1.1. The generic and concurrent versions of the collections are to be preferred because of their greater type safety and other improvements.
```

Comment:
I might consider reversing the last two sentences: Say what to do before saying what to avoid.